DeAceroDLabs/marketplace-frontend
# Feature/dmp 417

## What is done

[[DMP-417] Loading state cards](https://deacero.atlassian.net/browse/DMP-417)

## How is done

- Added loading prop to  `Card` component that will be used in home and search page for loading the cards.
- Created loadingCard component.

## How to test

1. Run the project via `npm start`
2. You should see the Home Screen. There you will see how the cards are loaded.
3. Type "clavo" to search bar and see the cards being load in search page.
